src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/validering/BarnetsAlderVilkårValidatorTest.kt

```diff
@@ -279,4 +281,50 @@ class BarnetsAlderVilkårValidatorTest {
         verify(exactly = 1) { barnetsAlderVilkårValidator2024.validerBarnetsAlderVilkår(vilkårResultatPerioder, person, any<LocalDate>(), any<LocalDate>()) }
         verify(exactly = 0) { barnetsAlderVilkårValidator2021og2024.validerBarnetsAlderVilkår(vilkårResultatPerioder, person, any<VilkårLovverkInformasjonForBarn>()) }
     }
+
+    @Test
+    fun `skal kalle på riktig valideringsfunksjon for adopsjonsbarn født så det fra fødselsdato ville ha vært truffet av både lovverk 2021 og 2024`() {
```

Comment:
Når denne testen kjøres i main blir validatoren for 2021 og 2024 kalt, så det verifiserer at problemet nå er løst 👍